base/random.jl

```diff
@@ -1096,6 +1096,14 @@ const ziggurat_nor_r      = 3.6541528853610087963519472518
 const ziggurat_nor_inv_r  = inv(ziggurat_nor_r)
 const ziggurat_exp_r      = 7.6971174701310497140446280481
 
+"""
+    randn([rng], [T=Float64], [dims...])
+
+Generate a normally-distributed random number of type `T` with mean 0 and standard deviation 1.
+Optionally generate an array of normally-distributed random numbers.
+The `Base` module currently provides an implementation for the types
+`Float16`, `Float32`, and `Float64` (the default).
+"""
```

Comment:
Oh, this was for using `fill-paragraph` in emacs without it touching the function body, and then forgot to remove the space.